pywren/cloudpickle/preinstalls.py

```diff
@@ -316,5 +316,11 @@
 ('scipy', False), 
 ('sklearn', False), 
 ('cvxpy', False), 
-('cvxopt', False)
+('cvxopt', False),
```

Comment:
Yeah, I think that's a good plan, although I think in the very near future this will be replaced / obsolete by the introspection stuff